docker-images/bundle/Makefile

```diff
@@ -0,0 +1,70 @@
+	$(CP) $(CRD_DIR)/033-ClusterRole-strimzi-kafka-client.yaml $(MANIFESTS)/strimzikafkaclient.clusterrole.yaml
+	$(CP) $(CRD_DIR)/050-ConfigMap-strimzi-cluster-operator.yaml $(MANIFESTS)/strimziclusteroperator.configmap.yaml
+
+	# Update exisiting Cluster Service Version (CSV) file
+	yq ea -i 'select(fi==0).spec.install.spec.deployments[0].spec = select(fi==1).spec | select(fi==0)' $(CSV_FILE) $(DEPLOYMENT_FILE)
+	yq ea -i 'select(fi==0).spec.install.spec.permissions[0].rules = select(fi==1).rules | select(fi==0)' $(CSV_FILE) $(CRD_DIR)/020-ClusterRole-strimzi-cluster-operator-role.yaml
+	yq ea -i 'select(fi==0).spec.install.spec.permissions[0].rules = select(fi==1).rules | select(fi==0)' $(CSV_FILE) $(CRD_DIR)/021-ClusterRole-strimzi-cluster-operator-role.yaml
+
+	yq ea -i '.metadata.name = "strimzi-cluster-operator-v$(CURRENT_BUNDLE_VERSION)"' $(CSV_FILE)
+	yq ea -i '.spec.install.spec.deployments[0].name = "strimzi-cluster-operator-v$(CURRENT_BUNDLE_VERSION)"' $(CSV_FILE)
+	yq ea -i '.spec.version = "$(CURRENT_BUNDLE_VERSION)"' $(CSV_FILE)
+	yq ea -i '.spec.replaces = "$(PREVIOUS_BUNDLE_VERSION)" | .spec.replaces style=""' $(CSV_FILE)
+	yq ea -i '.metadata.createdAt = "$(DATE)"' $(CSV_FILE)
+
+	# Update relatedImages section  
+	$(eval CLUSTER_OPERATOR_IMAGE=$(shell yq eval '.. | select(has("image")) | (select (.name == "*-operator")).image' $(DEPLOYMENT_FILE)))
+	$(eval KAFKA_BRIDGE=$(shell yq eval '.. | select(has("name")).env[] | (select (.name == "STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE")).value' $(DEPLOYMENT_FILE)))
+	$(eval JMXTRANS_IMAGE=$(shell yq eval '.. | select(has("name")).env[] | (select (.name == "STRIMZI_DEFAULT_JMXTRANS_IMAGE")).value' $(DEPLOYMENT_FILE)))
+	$(eval KANIKO_EXECUTOR_IMAGE=$(shell yq eval '.. | select(has("name")).env[] | (select (.name == "STRIMZI_DEFAULT_KANIKO_EXECUTOR_IMAGE")).value' $(DEPLOYMENT_FILE)))
+	$(eval KAFKA_IMAGE_VALUES=$(shell yq eval '.. | select(has("name")).env[] | (select (.name == "STRIMZI_KAFKA_IMAGES")).value' $(CSV_FILE)))
+
+	yq ea -i '.spec.relatedImages  = [{"name": "strimzi-cluster-operator", "value": "$(CLUSTER_OPERATOR_IMAGE)"}]' $(CSV_FILE)
+
+	# Add Kafka images to relatedImages section. Since make isolates each line into its own subshell,
+	# we run following shell commands as one line, joining the lines with ";" and "\"
+	for val in $(KAFKA_IMAGE_VALUES); \
+	do \
+		VERSION=`echo "$${val}" | cut -d'=' -f1 | tr -d '.'`;\
+		IMAGE=`echo "$${val}" | cut -d'=' -f2`;\
+		NAME="strimzi-kafka-$${VERSION}"\
+		name=$${NAME} image="$${IMAGE}" yq ea -i '.spec.relatedImages += [{"name": env(name), "value": env(image)}]' $(CSV_FILE); \
+	done\
+	
+	yq ea -i '.spec.relatedImages += [{"name": "strimzi-bridge", "value": "$(KAFKA_BRIDGE)"}]' $(CSV_FILE)
+	yq ea -i '.spec.relatedImages += [{"name": "strimzi-jmxtrans", "value": "$(JMXTRANS_IMAGE)"}]' $(CSV_FILE)
+	yq ea -i '.spec.relatedImages += [{"name": "strimzi-kaniko-executor", "value": "$(KANIKO_EXECUTOR_IMAGE)"}]' $(CSV_FILE)
```

Comment:
Any chance we can instead get the digest from the just built images? I assume the digest does not change by re-tagging etc., or?